keybase/platform_windows.go

```diff
@@ -58,9 +63,68 @@ func (c context) BeforeUpdatePrompt(update updater.Update, options updater.Updat
 	return nil
 }
 
+func (c config) promptProgram() (command.Program, error) {
+	destinationPath := c.destinationPath()
+	if destinationPath == "" {
+		return command.Program{}, fmt.Errorf("No destination path")
+	}
+
+	return command.Program{
+		Path: "mshta.exe",
+		Args: []string{filepath.Join(destinationPath, "prompter", "prompter.hta")},
+	}, nil
+}
+
 func (c context) UpdatePrompt(update updater.Update, options updater.UpdateOptions, promptOptions updater.UpdatePromptOptions) (*updater.UpdatePromptResponse, error) {
-	// No update prompt for Windows, since the installer may handle it
-	return &updater.UpdatePromptResponse{Action: updater.UpdateActionContinue}, nil
+	promptProgram, err := c.config.promptProgram()
+	if err != nil {
+		return nil, err
+	}
+
+	promptJSONInput, err := c.promptInput(update, options, promptOptions)
```

Comment:
I would put OutPath in promptOptions... edit this method to handle that and then you don't need extra OutPath stuff below